l10n_br_mis_builder_cash_flow_sale/models/sale_order.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2024 - TODAY, Kaynnan Lemes <kaynnan.lemes@escodoo.com.br>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class SaleOrder(models.Model):
+
+    _inherit = "sale.order"
+
+    @api.depends(
+        "partner_id",
+        "amount_total",
+        "amount_financial_total",
+        "pricelist_id",
+        "currency_id",
+        "order_line.invoice_lines",
+        "order_line.qty_invoiced",
+        "invoice_ids",
+    )
+    def _compute_forecast_uninvoiced_amount(self):
+        super()._compute_forecast_uninvoiced_amount()
+        for order in self:
```

Comment:
aqui soh rodaria se a empresa do registro for do brasil.. temos exemplos assim na localização

Reply:
Uma validação antes então da company né?